test/format/format.h

```diff
@@ -95,16 +97,18 @@ typedef struct {
     char *home_rand;     /* RNG log file path */
     char *home_stats;    /* Statistics file path */
 
-    char *config_open;                     /* Command-line configuration */
-    char wiredtiger_open_config[8 * 1024]; /* Database open config */
+    char *config_open; /* Command-line configuration */
 
     bool rand_log_stop; /* Logging turned off */
     FILE *randfp;       /* Random number log */
 
     uint32_t run_cnt; /* Run counter */
 
-    bool logging; /* log operations  */
-    FILE *logfp;  /* log file */
+    bool logging;              /* log operations  */
+    bool log_all;              /* log all operations  */
+    bool log_local;            /* log to the primary database */
+    WT_CONNECTION *oplog_conn; /* optional logging database */
+    WT_SESSION *oplog_session;
```

Comment:
can we call this something other than oplog? MongoDB has an oplog and many internal WiredTiger testing applications simulate the oplog behavior and call that "oplog". Using a different name here will help preserve my sanity.

Reply:
@agorrod, 
> can we call this something other than oplog?

Good point, I'll switch to "trace".